cmd/cmd_pkg.go

```diff
@@ -623,7 +624,7 @@ func pushCommandHandler(cmd *cobra.Command, args []string) {
 	}
 	defer os.RemoveAll(archiveName)
 
-	req, err := api.BuildRequestForArchiveUpload(ns, name, foundPkg, archiveName, private)
+	req, err := api.BuildRequestForArchiveUpload(ns, name, foundPkg, archiveName, private, arch)
```

Comment:
can you change this to pkgFlags.Parch() ?

then I believe you can get rid of the arch flag above as well

Reply:
Oh shoot I just saw this, is the pr still relevant? I can sort it out 😃

Reply:
yeh - i'd just like to have everything use that one Parch() method, and then you can drop that other arch flag

Reply:
Ok I'll do that when I'm by my laptop